### PR TITLE
add gce instance script

### PR DESCRIPTION
**Pull request by davidzhu22**
add gce bash script to create instance, delete instance.
add install fornaxcore and nodeagent script.

### CODE REVIEW

File: hack/test-gce/nodeagent_install.sh [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+	   echo export GOROOT=\"/usr/local/go\" >> ~/.bashrc
+	   echo export GOPATH=\"\$HOME/go\" >> ~/.bashrc
+	   echo export GOBIN=\"\$HOME/go/bin\" >> ~/.bashrc
+	   echo export PATH=\"/usr/local/go/bin:\$HOME/go/bin:\$PATH\" >> ~/.bashrc
+       source $HOME/.bashrc
+       echo -e "## DONE\n"
+    else
+       echo -e "## go${GO_VERSION} already installed\n "
+   fi
+}
```
Comment (guowenjian90) [inline hack/test-gce/nodeagent_install.sh:214]: we do not need to install development tools, you should just scp local image into VM

File: hack/test-gce/nodeagent_install.sh [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+nodeagent_build(){
+    echo -e "## CLONE FORNAXCORE SOURCE CODE"
+    cd go
+	mkdir -p bin src pkg
+	cd src
+	mkdir -p centaurusinfra.io
+	cd centaurusinfra.io
+	# pushd $HOME/go/src/centaurusinfra.io
+    sudo git clone https://github.com/CentaurusInfra/fornax-serverless.git
```
Comment (guowenjian90) [inline hack/test-gce/nodeagent_install.sh:225]: we do not need to install development tools and build source code in VM, you should just scp local image into VM

File: hack/test-gce/gce_create.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+# davidzhu-nodeagent2
+gcloud compute instances create davidzhu-nodeagent2 --zone=us-central1-a --machine-type=e2-medium --network-interface=network-tier=PREMIUM,subnet=default --maintenance-policy=MIGRATE --provisioning-model=STANDARD --service-account=[your_account_number]-compute@developer.gserviceaccount.com --scopes=https://www.googleapis.com/auth/cloud-platform --tags=http-server,https-server --create-disk=auto-delete=yes,boot=yes,device-name=davidzhu-instance-1,image=projects/ubuntu-os-cloud/global/images/ubuntu-2004-focal-v20220927,mode=rw,size=50,type=projects/quark-serverless/zones/us-central1-a/diskTypes/pd-ssd --no-shielded-secure-boot --shielded-vtpm --shielded-integrity-monitoring --reservation-affinity=any
```
[the remark this answers is not in the document]
Comment (davidzhu22) [reply hack/test-gce/gce_create.sh:12]: Yes, I will work on it by using a for loop and read command to pass the parameter

File: hack/test-gce/gce_delete.sh [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+set -e
+
+# delete virtual machine instance
+gcloud compute instances delete davidzhu-fornaxcore --zone=us-central1-a
+gcloud compute instances delete davidzhu-instance-1 --zone=us-central1-a
+gcloud compute instances delete davidzhu-instance-2 --zone=us-central1-a
```
[the remark this answers is not in the document]
Comment (davidzhu22) [reply hack/test-gce/gce_delete.sh:8]: Yes, I will work on it by using a for loop and read command to pass the parameter

File: hack/test-gce/nodeagent_install.sh (hunk 2 above)
Comment (davidzhu22) [reply hack/test-gce/nodeagent_install.sh:225]: Let me try to direct copy the already build exe file to nodeagent

**Pull request closed without merging**